Mittons.Fixtures/Extensions/OptionExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Mittons.Fixtures.Models;
+
+namespace Mittons.Fixtures.Extensions
+{
+    /// <summary>
+    /// Extensions methods for <see cref="Option"/>.
+    /// </summary>
+    public static class OptionExtensions
+    {
+        /// <summary>
+        /// Gets the option as a string formatted for use in a docker command.
+        /// </summary>
+        /// <param name="option"></param>
+        /// <returns>The option as a string formatted for use in a docker command.</returns>
+        /// <exception cref="ArgumentNullException">Thrown when the option, or its properties, are <c>null</c>.</exception>
+        public static string ToCommandFormattedString(this Option option)
+        {
+            if(option == null)
+            {
+                throw new ArgumentNullException(nameof(option));
+            }
+
+            if(option.Name == null)
+            {
+                throw new ArgumentNullException(nameof(option.Name));
+            }
+
+            if(option.Value == null)
```

Comment:
In the code before we had it defaulting to `string.Empty` so that led me down the path that is just wouldn't be nullable. Handling `null` will hopefully be easier as it wouldn't even be an option.
That said, if you're thinking that `string.Empty` of `" "` (whitespace) shouldn't be allowed either then I could see that being logic we could strive for.

Reply:
The point is, though, that I don't think we should be throwing an exception in these cases? How are you expecting to support options without values?

Reply:
Wanted to make sure that's the point you were trying to get across, apologies.
Empty string is how it seemed to be handled prior to the changes, right? I figured we may want to keep that -- as it was, and for the sake of avoiding nullable sprawl.

So, if we would continue as is, `null` isn't valid but empty string (or whitespace) would be allowed through. That could be something placed into the remarks in the code documentation if we'd want?

If you're leaning towards just skipping the null check and instead treating null, empty, and whitespace as "empty" then I can make that change for us :).

Reply:
I added to the code docs a tad for this, but only reflecting what it was staying inline with. Let me know if you're thinking of a different direction for it.

Reply:
@Frueber sorry, just saw this. Yeah, I'd rather treat all 3 the same. If anything, null seems like the _most_ applicable in this situation. I'm thinking what might make sense is to drop the check, but change the return to this:
return string.IsNullOrWhitespace(option.Value) ? option.Name : $"{option.Name} \"{option.Value\"";

Reply:
Updated as suggested. `null`, `empty string`, and `white-space` will be handled as just the option name, else the option name with a value.